assets/js/blocks/collection-filters/utils.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import { ProductCollectionQuery } from '@woocommerce/blocks/product-collection/types';
+
+function taxonomyMapper( taxonomy: string ) {
```

Comment:
Usual Lucio nitpick… usually you see a noun used to name a `class` rather than a `function`. So, I would expect a `TaxonomyMapper` class, but I'd use a verb to name a function: e.g. `mapTaxonomy`.